content/docs/install/plugins.md

```diff
@@ -6,13 +6,44 @@ your data and transformations.
 
 The file itself is in a simple YAML format.
 
+We also provide a [schema](https://github.com/iterative/dvcyaml-schema) for
+dvc.yaml that provides automatic syntax checks and auto-completion.
+
+## Visual Studio Code
+
+To make `.dvc` files and `dvc.lock` files as YAML, you should add following in
+the settings:
+
+```json
+"files.associations": {
+    "*.dvc": "yaml",
+    "dvc.lock": "yaml"
+}
+```
+
+To get auto-completion and syntax checks, you can install
+[YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml)
+that provides auto-completion and syntax checks for `dvc.yaml` files.
+
+## Pycharm/Intellij
+
+Pycharm/Intellij should ask how to associate file types for `.dvc` and
+`dvc.lock` files when you first open them. If not, follow
+[Pycharm's guide](https://www.jetbrains.com/help/pycharm/creating-and-registering-file-types.html)
+to set them manually.
+
+Pycharm, by default, provides auto-completion and syntax checks for `dvc.yaml`
```

Comment:
Would be good to confirm this. For me, it worked without any change in settings.

Reply:
I installed the free community edition on Windows and it did work indeed but only after a long indexing process.

Reply:
You should have opened `example-get-started`, would have been faster.

Reply:
The first time PyCharm indexes, it goes over the entire Python stdlib I think haha.